### PR TITLE
Fix indentation issue after COMMA token

### LINKED ISSUE #336: Module attributes with a comma break indentation for the rest of the file (sometimes)
The following code breaks the indentation for the rest of the file:

```elixir
defmodule TestIndentation do
  import Enum
  @att %{ab: "cd",
         de: "fg"}
| <-- cursor is here
end
```

If I remove the `import Enum` line or the line-break after the comma, everything is fine.

Please let me know if I can help narrow this down further. I'm using spacemacs and everything is up to date, if that's relevant.

### PR DESCRIPTION
**Pull request by tonini**
fixes #336

### CODE REVIEW

File: test/elixir-mode-indentation-test.el [hunk 1]
```diff
@@ -1947,6 +1947,25 @@ defmodule Test do
 end
 ")
 
+(elixir-def-indentation-test indent-before-comma
+                             (:tags '(indentation))
+"
+defmodule TestIndentation do
+import Enum
+
```
Comment (utkarshkukreti) [inline test/elixir-mode-indentation-test.el]: The problem actually occurred only when there was no newline here (after `import Enum`).

Comment (utkarshkukreti) [inline test/elixir-mode-indentation-test.el]: https://github.com/elixir-lang/emacs-elixir/issues/336#issuecomment-211770842

Comment (tonini) [inline test/elixir-mode-indentation-test.el]: Its working both ways, Its tested but you're free to add another test case too

Comment (utkarshkukreti) [inline test/elixir-mode-indentation-test.el]: Yes, just tested, it works :)

**Pull request merged**